src/widgets/home-page-info/HomePageInfo.vue

```diff
@@ -99,8 +99,12 @@ const nodesAmount = ref(0);
 
 const { handleUnknownError } = useErrorHandlers();
 
+const isLoading = ref(false);
```

Comment:
Replace with `usePromise` or `useTask`.

The point of this PR is to reduce this repetitive manual data-fetching related reactivity logic.